apps/docs/content/docs/guide/installation.mdx

```diff
@@ -152,6 +152,26 @@ If you are using pnpm, you need to add the following line to your `.npmrc` file
 public-hoist-pattern[]=*@heroui/*
 ```
 
+<Blockquote color="primary">
+**Note**: In version <10 of pnpm the default values of the hoist pattern contains ['*eslint*', '*prettier*'] and by adding the hoist pattern they will get removed and you may experince the tools not working.
```

Comment:
This is originally a pnpm issue, so I'm not sure if we should include setup details for external libraries like Prettier or ESLint.Providing useful information is important, but we should also avoid making the documentation overly lengthy. I'd appreciate hearing other members' thoughts.